src/components/Create/index.tsx

```diff
@@ -726,7 +726,7 @@ export default function Create() {
           }
           onOk={deployProject}
           confirmLoading={loadingCreate}
-          width={800}
+          width={600}
```

Comment:
what's the reason behind this change? the modal used to be narrower (i think 600) but was widened mainly because when things like website + pay text are set it, can run long and look awkward

i'd be happy to work some reformatting into this PR though if you have some ideas

Reply:
I initially dropped this down because it seemed like there was a lot of unnecessary whitespace when some fields were empty. But now that they're spread out more horizontally I do think 800px actually looks better.